src/entities/raw-data-providers/jormungandr-api.js

```diff
@@ -39,12 +44,30 @@ class JormungandrApi implements RawDataProvider {
     this.#networkBaseUrl = urljoin(network.bridgeUrl, 'api/v0')
     this.#parser = parser
     this.logger = logger
+    this.retryCount = retryCount || -1
     this.networkSlotsPerEpoch = networkConfig.slotsPerEpoch()
   }
 
+  async getWithRetry(path: string, options?: {}) {
+    for (let alreadyTriedCount = 1; ;alreadyTriedCount++) {
+      try {
+        const resp = await this.get(path, options)
+        return resp
+      } catch (e) {
+        const millisSleep = alreadyTriedCount * MILLIS_RETRY
```

Comment:
are we okay with this maybe starting to sleep for a really long amount of time? Seems like there should be a max. Honestly you could even just use an exponential backoff library instead of re-implementing this yourself if you want